deploy/template/etc/init/docker-worker.conf

```diff
@@ -18,11 +18,14 @@ script
     . /etc/default/$UPSTART_JOB
   fi
 
-  # Pull images used for sidecar containers
-  docker pull taskcluster/taskcluster-proxy:latest
-  docker pull taskcluster/livelog:v4
-  docker pull taskcluster/dind-service:v4.0
-  docker pull taskcluster/relengapi-proxy:2.0.1
+  echo "Loading docker sidecar images"
```

Comment:
I see, so the `docker pull` commands removed in this hunk sometimes failed, but `docker-worker.conf` continued executing and started the worker, which then tried to pull again, and failed again.